test/lit.cfg

```diff
@@ -295,6 +295,10 @@ sdk_overlay_linker_opt = ""
 sdk_overlay_dir_opt = ""
 test_sdk_overlay_dir = lit_config.params.get('test_sdk_overlay_dir', None)
 if test_sdk_overlay_dir is not None:
+
+    # TODO: Is this the right place to put this?
+    config.available_features.add('sdk_overlay')
```

Comment:
@gottesmm Is this the right place to add the `sdk_overlay` feature?

Reply:
What's blocking this PR from being merged?

Reply:
I suppose that's the right place for this. I'll remove the comment and merge.
Ideally we can pair this with a build system unit test. @Rostepher is that still a possibility?

Reply:
I'm not sure I understand what you mean by a build-system unit test in this context? What behavior do you want to test?

Reply:
@harlanhaskins – Based on @Rostepher's reply, what do you think about merging this now and worrying about how to unit test the build script later?

Reply:
I'll go ahead and merge it and follow up offline with @Rostepher about how to build this unit test.